unity/Holo/Assets/Scripts/model_with_plate/ModelWithPlate.cs

```diff
@@ -1,24 +1,81 @@
-﻿using UnityEngine;
+﻿using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEngine;
+
 using HoloToolkit.Unity;
 using HoloToolkit.Unity.UX;
+using HoloToolkit.Unity.Buttons;
+using HoloToolkit.Unity.InputModule;
+using System.Text;
 
 public class ModelWithPlate : MonoBehaviour, IClickHandler
 {
     /* Public fields that should be set in Unity Editor */
-    public TextMesh PlayOrStopText;
+    public CompoundButtonText PlayOrStopText;
     public GameObject ButtonsModel;
     public GameObject ButtonsModelPreview;
     public GameObject PlateAnimated;
+    public Material MaterialPreview;
+    public Material MaterialNonPreview;
 
-    private GameObject collectionButtons;
+    public GameObject ModelClipPlane;
+
+    ModelClippingPlaneControl ModelClipPlaneCtrl;
+    HandDraggable HandDraggableTranslation;
 

```

Comment:
The code of `ModelWithPlate.cs` changed in master a bit, so it'll be easier to see the changes after rebasing this PR on latest master :) I added HandDraggable component handling to ModelWithPlate.cs in PR #25 , so this PR doesn't need to do this.